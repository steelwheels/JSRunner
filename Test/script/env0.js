
function main(args)
{
	let pwd = Environment.get("PWD") ;
	console.log("pwd is string => " + isString(pwd)) ;

	Environment.set("A", "1234") ;
	console.log("A = " + Environment.get("A")) ;

	let curdir = Environment.currentDirectory ;
	console.log("curdir is URL: " + isURL(curdir)) ;

	let tmpdir = Environment.temporaryDirectory ;
	console.log("tmpdir is URL: " + isURL(tmpdir)) ;

	return 0 ;
}

