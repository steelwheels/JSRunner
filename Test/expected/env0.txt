pwd is string => true
A = 1234
curdir is URL: true
tmpdir is URL: true
